contracts/cwd-core/src/msg.rs

```diff
@@ -139,78 +144,95 @@ pub enum ExecuteMsg {
 
 #[voting_query]
 #[info_query]
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-#[serde(rename_all = "snake_case")]
+#[cw_serde]
+#[derive(QueryResponses)]
 pub enum QueryMsg {
     /// Get's the DAO's admin. Returns `Addr`.
+    #[returns(Addr)]
     Admin {},
-    /// Get's the currently nominated admin (if any). Returns
-    /// `AdminNominationResponse`.
+    /// Get's the currently nominated admin (if any).
+    #[returns(AdminNominationResponse)]
```

Comment:
this should fix the name resolution lints you're getting

```suggestion
    #[returns(::query::AdminNominationResponse)]
```